ext/compressors/zstd/zstd_compress.c

```diff
@@ -89,6 +112,68 @@ zstd_error(WT_COMPRESSOR *compressor, WT_SESSION *session, const char *call, siz
     return (WT_ERROR);
 }
 
+/*
+ * zstd_alloc_context --
+ *     WiredTiger Zstd allocate context from context pool.
+ */
+static void
+zstd_alloc_context(
```

Comment:
I'd prefer to use the names `zstd_get_context` and `zstd_release_context`. I associate alloc/free with memory management - which this mechanism is aimed at avoiding.